<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkTimerWidget</class>
 <widget class="QWidget" name="WorkTimerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QVBoxLayout" name="mainContentLayout">
     <item>
      <layout class="QHBoxLayout" name="topLayout">
       <item>
        <widget class="QPushButton" name="toggle_settings">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/settings.svg</normaloff>
           <normalon>:/new/buttons/ui/settingsON.svg</normalon>:/new/buttons/ui/settings.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="theme_button">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/light.svg</normaloff>:/new/buttons/ui/light.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minimize_button">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/minimize.svg</normaloff>:/new/buttons/ui/minimize.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_button">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/exit.svg</normaloff>:/new/buttons/ui/exit.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="font">
        <font>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>25:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sessionLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сессия 1 - Рабочее время</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <spacer name="leftSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/play.svg</normaloff>:/new/buttons/ui/play.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/pause.svg</normaloff>:/new/buttons/ui/pause.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/stop.svg</normaloff>:/new/buttons/ui/stop.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restartButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/buttons/ui/restart.svg</normaloff>:/new/buttons/ui/restart.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="rightSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="settingsSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <widget class="QGroupBox" name="settingsGroup">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>0</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text" stdset="0">
    <string>Настройки</string>
   </property>
   <layout class="QVBoxLayout" name="settingsLayout">
    <item>
     <layout class="QGridLayout" name="settingsGrid">
      <item row="0" column="0">
       <widget class="QLabel" name="workDurationLabel">
        <property name="text">
         <string>Рабочее время</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="workTimeEdit">
        <property name="displayFormat">
         <string>H:mm:ss</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>25</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="shortBreakLabel">
        <property name="text">
         <string>Короткий перерыв</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="shortBreakTimeEdit">
        <property name="displayFormat">
         <string>H:mm:ss</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>5</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="longBreakLabel">
        <property name="text">
         <string>Длинный перерыв</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="longBreakTimeEdit">
        <property name="displayFormat">
         <string>H:mm:ss</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>15</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sessionsLabel">
        <property name="text">
         <string>Сессий до длинного перерыва</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sessionsSpinbox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="breakSoundLabel">
        <property name="text">
         <string>Звук перерыва</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="breakSoundCombo"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="workSoundLabel">
        <property name="text">
         <string>Звук работы</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="workSoundCombo"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="volumeLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources.qrc">:/new/buttons/ui/volume.svg</pixmap>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="volumeSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="checkboxLayout">
      <item>
       <widget class="QCheckBox" name="pinCheckbox">
        <property name="text">
         <string>Поверх всех окон</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="taskbarCheckbox">
        <property name="text">
         <string>Отображать в панели задач</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="languageSwitch">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>RU</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
